pulp_container/app/access_policy.py

```diff
@@ -19,12 +20,12 @@ def has_add_namespace_perms(self, request, view, action):
             return request.user.has_perm("container.add_containernamespace")
         else:
             return request.user.has_perm(
-                "container.view_containernamespace"
-            ) or request.user.has_perm("container.view_containernamespace", namespace)
+                "container.manage_namespace_distributions"
+            ) or request.user.has_perm("container.manage_namespace_distributions", namespace)
 
     def has_view_namespace_perms(self, request, view, action):
```

Comment:
@lubosmj is there any benefir of keeping this check for perm?

Reply:
@ipanova, I think we can get rid of that method.

Regarding the method `has_manage_namespace_distributions_perms()`, what do you think about providing an additional parameter `permission` that will be passed to `has_perm()`, like so: `request.user.has_perm(permission, namespace)`? When a user wants just to view a namespace, he or she does not need to have the `manage_namespace_distribution` permission, but only `view_containernamespace`.

Reply:
I have left this method to decouple calls that require object-level perms on the namespace to perform update/patch/delete on existing distribution and model/object-level perms on the namespace when creating a new distribution.
Maybe we can improve this later when having a full picture of all other rbac objects